Documentation/cmdref/cilium-agent_hive.md

```diff
@@ -25,6 +25,8 @@ cilium-agent hive [flags]
       --controller-group-metrics strings                          List of controller group names for which to to enable metrics. Accepts 'all' and 'none'. The set of controller group names available is not guaranteed to be stable between Cilium versions.
       --egress-gateway-policy-map-max int                         Maximum number of entries in egress gateway policy map (default 16384)
       --egress-gateway-reconciliation-trigger-interval duration   Time between triggers of egress gateway state reconciliations (default 1s)
+      --enable-bandwidth-manager                                  Enable BPF bandwidth manager
+      --enable-bbr                                                Enable BBR for the bandwidth manager
```

Comment:
That would be nice, but this flag isn't new. It appears that way because now that we provide it via hive it gets added to these docs, but this flag is already in use, so we can't just change it.